elevate-cpanel

```diff
@@ -406,8 +430,13 @@ sub upgrade_to_rocky ($self) {
     return $self->upgrade_to() eq UPGRADE_TO_ROCKY;
 }
 
+sub upgrade_to_cloudlinux ($self) {
+    return $self->upgrade_to() eq UPGRADE_TO_CLOUDLINUX;
+}
+
 sub upgrade_to_pretty_name ($self) {    # used by output messages
```

Comment:
just this now.
```
return Cpanel::OS::pretty_distro() . ' 7';
```

Reply:
`upgrade_to_pretty_name`, you mean? Why 7?

Reply:
I mean all of those can be replaced with the one line.